int32 actionId
int32 actionSubId
int32 armId
---

